content/sensu-go/5.16/guides/email-handler.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Send email alerts with the Sensu Go Email Handler"
+linkTitle: "Send Email Alerts"
+description: "Here’s how to send alerts to your email with the Sensu Go Email Handler. Use handlers to send events to your technology of choice (in this case, email) to alert you of incidents and help you resolve them more quickly."
+weight: 85
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+- [Create a check](#create-a-check)
+- [Add the email handler asset](#add-the-email-handler-asset)
+- [Add an event filter](#add-an-event-filter)
+- [Assign the email handler to a check](#assign-the-email-handler-to-a-check)
+- [Create an email template](#create-an-email-template)
+- [Next steps](#next-steps)
+
+Sensu event handlers are actions the Sensu backend executes on [events][1].
+This guide explains how to use the [Sensu Go Email Handler][3] to send incident notification emails.
```

Comment:
I'm a little torn on using 'incident' here. IMO it's a bit of a laden term. My suggest is to just say `send notifications emails`.

```suggestion
This guide explains how to use the [Sensu Go Email Handler][3] to send notification emails.
```